src/scripts/main.js

```diff
@@ -1,7 +1,115 @@
 'use strict';
 
 // Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+import Game from '../modules/Game.class.js';
+
+const game = new Game();
 
 // Write your code here
+const gameScore = document.querySelector('.game-score');
+const startButton = document.querySelector('.buttom start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const messageStart = document.querySelector('message-start');
+
+const fields = document.querySelector('.field-cell');
```

Comment:
The selector '.field-cell' should likely select multiple elements. Consider using `document.querySelectorAll` instead of `document.querySelector`.